Bio/SCOP/__init__.py

```diff
@@ -592,7 +594,9 @@ def getDescendents(self, node_type):
         return nodes
 
     def getAscendent(self, node_type):
-        """Return the ancenstor node of the given type, or None.Node type can a
+        """Return the ancenstor node of the given type, or None.
+
+        Node type can a
```

Comment:
This (and the original) are missing the word "be" here, and the line wrapping looks odd. The ``getDescendents`` text is fine.